docs/resources/cli-reference.md

```diff
@@ -175,6 +175,7 @@ launchable subset [OPTIONS] TESTRUNNER ...
 | `--flavor KEY=VALUE`      | Advanced option. Restrict the subset of tests by `flavor`. Flavors must be submitted ahead of time with test reports (see `launchable record tests --flavor` below). Supports multiples, e.g. `--flavor key1=value1 --flavor key2=value2`.                                                                                                                                                                                                                                                                                                | No                                                        |
 | `--rest FILE`             | Output the remainder of the subset to a file. This is useful for running the "rest of the tests" after you've run a subset.                                                                                                                                                                                                                                                                                                                                                                                                               | No                                                        |
 | `--split`                 | Output a subset ID instead of the subset list itself. For use with `launchable split-subset`                                                                                                                                                                                                                                                                                                                                                                                                                                              | No                                                        |
+| `--ignore-new-tests`      | Ignore that were added recently. NOTICE: this option will ignore tests that you added just now, too.                                                                                                                                                                                                                                                                                                                                                                                                                                      | No                                                        |
```

Comment:
[grammer]
`Ignore tests that were added recently. NOTICE: this option will ignore tests that you added just now as well.`

Reply:
Ops... Thank you!